packages/@react-aria/tag/docs/useTagGroup.mdx

```diff
@@ -0,0 +1,297 @@
+* Labeling support for accessibility
+* Support for mouse, touch, and keyboard interactions
+
+## Anatomy
+
+<Anatomy />
+
+A tag group consists of a list of tags.
+If a visual label is not provided, then an `aria-label` or
+`aria-labelledby` prop must be passed to identify the tag group to assistive technology.
+
+Individual tags should include a visual label, and may optionally include icons or a clear button.
+
+`useTagGroup` returns props for the group and its label, which you should spread
+onto the appropriate element:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTagGroup.return.id].properties} />
+</TypeContext.Provider>
+
+`useTag` returns props for an individual tag:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useTag.return.id].properties} />
+</TypeContext.Provider>
+
+In order to be correctly identified to assistive technologies and enable proper keyboard navigation, the tag group should use `gridProps` on its outer container.
+
+Each individual tag should use `rowProps` on its outer container, and use `gridCellProps` on an inner container.
+
+## Example
+
+```tsx example export=true
+import {useTag, useTagGroup} from '@react-aria/tag';
+import {useTagGroupState} from '@react-stately/tag';
+import {Item} from '@react-stately/collections';
+import {useFocusRing} from '@react-aria/focus';
+
+// Reuse the Button from your component library. See below for details.
+import {Button} from 'your-component-library';
+
+function TagGroup(props) {
+  let { label, description, errorMessage, validationState, allowsRemoving, onRemove } = props;
+  let ref = React.useRef(null);
+
+  let state = useTagGroupState(props);
+  let {
+    gridProps,
+    labelProps,
+    descriptionProps,
+    errorMessageProps
+  } = useTagGroup(props, state, ref);
+
+  return (
+    <div ref={ref} className="tag-group">
+      <div {...labelProps}>{label}</div>
+      <div {...gridProps}>
+        {[...state.collection].map((item) => (
+          <Tag
+            {...item.props}
+            key={item.key}
+            item={item}
+            state={state}
+            allowsRemoving={allowsRemoving}
+            onRemove={onRemove}
+          >
+            {item.rendered}
+          </Tag>
+        ))}
+      </div>
+      {description && (
+        <div {...descriptionProps} className="description">
+          {description}
+        </div>
+      )}
+      {errorMessage && validationState === "invalid" && (
+        <div {...errorMessageProps} className="error-message">
+          {errorMessage}
+        </div>
+      )}
+    </div>
+  );
+}
+
+function Tag(props) {
+  let { item, state, allowsRemoving, onRemove } = props;
+  let ref = React.useRef(null);
+  let { isFocused, focusProps } = useFocusRing({
+    within: true
+  });
+
+  let { clearButtonProps, labelProps, gridCellProps, rowProps } = useTag(
```

Comment:
I noticed that `useTagGroup` above lists the return props in the same order as they are listed in the `Anatomy` section. Here `clearButtonProps` of `useTag` is first instead of last, like it would be if it mirrored the order from the `Anatomy` section.